backend/routes/PostReaction/ShareHandler.py

```
@@ -0,0 +1,93 @@
+
+def get_media_data(db: Session, post: Post):
+    """Get media-specific post data"""
+    media = db.query(PostMedia).filter(PostMedia.post_id == post.id).first()
+    return {"media_url": f"http://127.0.0.1:8000/uploads/media/{media.media_url}" if media else None}
+
+def get_document_data(db: Session, post: Post):
+    """Get document-specific post data"""
+    document = db.query(PostDocument).filter(PostDocument.post_id == post.id).first()
+    return {"document_url": f"http://127.0.0.1:8000/uploads/document/{document.document_url}" if document else None}
+
+def get_event_data(db: Session, post: Post):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_event_data`

(ANN201)